scripts/byoc-e2e-probe.py

```
@@ -0,0 +1,82 @@
+)
+DMZ = os.environ.get("BYOC_SIGNER_URL", "https://pymthouse-production.up.railway.app")
+ORCH = os.environ.get("BYOC_ORCH_URL", "https://byoc-staging-1.daydream.monster:8935")
+CAP = os.environ.get("BYOC_CAPABILITY", "flux-schnell")
+
+
+def _post(url: str, body: dict, headers: dict | None = None) -> tuple[int, str]:
+    data = json.dumps(body).encode()
+    hdrs = {"Content-Type": "application/json", **(headers or {})}
+    req = urllib.request.Request(url, data=data, headers=hdrs, method="POST")
+    try:
+        with urllib.request.urlopen(req, timeout=60) as resp:
```

Ruff (0.15.21)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)